### DISCUSSION TITLE
Updating Services

### DISCUSSION THREAD

**Discussion by 0xcadams** [obsproject/obs-studio#9127] [Development Questions] [answered]
Hi, I have a question about updating the services in OBS - we submitted a PR to add a service and haven't yet seen it propagate to OBS. We followed the service submission guidelines [1] and don't see any obvious errors in CI. Are there any further steps we have to take to publish the latest services.json version?

[1] https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines

**Comment by Fenrirthviti** [comment, marked as answer]
Service updates are deployed independently of OBS versions, but it is currently a manual process. We are working to improve the deployment process for this, as it currently only happens about once a month or so.
